ngraph/src/ngraph/op/non_max_suppression.hpp

```diff
@@ -177,8 +177,7 @@ namespace ngraph
                 BoxEncodingType m_box_encoding = BoxEncodingType::CORNER;
                 bool m_sort_result_descending = true;
                 ngraph::element::Type m_output_type = ngraph::element::i64;
-
-            private:
```

Comment:
Is it expected that you made `max_boxes_output_from_input` `public`?

Reply:
I have made it `protected` (not `public`) so I can use it in the sub-classes.

Reply:
Thanks, for clarification